ddocs/erlang_filters/packages/traditional-couchapp/build/load.js

```diff
@@ -0,0 +1,77 @@
+var fs = require('fs')
```

Comment:
Could you add a documentation file to the traditional-couchapp dir, to explain what is going on? (or elsewhere, or in this file, wherever works)
If you die, will we be (blinded by grief) able to know what's going on?